pom.xml

```diff
@@ -44,8 +44,15 @@
         </pluginRepository>
     </pluginRepositories>
 
+    <!-- Other properties such as kafka.version are derived from parent project(s) such as
+         https://github.com/confluentinc/common (see common's pom.xml for kafka.version).
+    -->
     <properties>
-        <algebird.version>0.13.0</algebird.version>
+        <!-- Only required because confluentinc/common currently sets this to 1.2.0-SNAPSHOT,
+             and the Scala facade for Kafka Streams is only in 2.0.0-SNAPSHOT and higher. -->
+        <kafka.version>2.0.0-SNAPSHOT</kafka.version>
```

Comment:
This should be removed once `confluentinc/common` et al. set `kafka.version` to `2.0.0-SNAPSHOT` (this should happen very soon).